misk/src/main/kotlin/misk/web/interceptors/MetricsInterceptor.kt

```diff
@@ -20,10 +20,10 @@ internal class MetricsInterceptor internal constructor(
     val (elapsedTime, result) = timed { chain.proceed(chain.httpCall) }
 
     val elapsedTimeMillis = elapsedTime.toMillis().toDouble()
-    val callingPrincipal = caller.get()?.principal ?: "unknown"
+    val callingService = caller.get()?.service ?: "unknown"
```

Comment:
add a test that creates a request with `USER_HEADER` set and show that unknown is set in the metric?